discretisedfield/tests/test_field.py

```diff
@@ -2933,6 +2933,22 @@ def test_rfft_no_shift_last_dim():
     assert np.array_equal(field_ft.array[..., 0], ft)
 
 
+def test_1d_fft():
+    mesh = df.Mesh(p1=0, p2=10, cell=2)
+    f = mesh.coordinate_field()
+    field_ft = f.fftn()
+    expected_array = np.array(
+        [
+            -5.0 - 1.62459848j,
+            -5.0 - 6.8819096j,
+            25.0 - 0.0j,
+            -5.0 + 6.8819096j,
+            -5.0 + 1.62459848j,
+        ]
+    )
```

Comment:
How did you compute the reference values?

Reply:
I used `numpy.fft` as we use scipy in the code

Reply:
Should we then maybe just add this to the test instead of the hard-coded values? We are anyways not interested in precisely testing the values and instead mainly want to ensure that we compute something reasonable (i.e. an FFT at all).